BUG: SCA 2021.x build failed

**Which Component**
spring cloud alibaba 2021 branch

**Describe the bug**
Cloning the project to the local, executing the maven command to build failed

**To Reproduce**
1. git pull https://github.com/alibaba/spring-cloud-alibaba.git 2021.x:master
2. mvn clean install -DskipTests=true

**Expected behavior**
build success

**stack trace**
```markdown
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  43.967 s
[INFO] Finished at: 2023-06-08T14:08:03+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project spring-cloud-
starter-alibaba-nacos-config: Compilation failure: Compilation failure: 
[ERROR] /C:/Users/14815/Desktop/sca2021/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alib
aba/cloud/nacos/NacosConfigProperties.java:[30,24] 程序包javax.annotation不存在
[ERROR] /C:/Users/14815/Desktop/sca2021/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alib
aba/cloud/nacos/NacosConfigProperties.java:[101,10] 找不到符号
[ERROR]   符号:   类 PostConstruct
[ERROR]   位置: 类 com.alibaba.cloud.nacos.NacosConfigProperties
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :spring-cloud-starter-alibaba-nacos-config
```

--- a/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/pom.xml
+++ b/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/pom.xml
@@ -67,6 +67,12 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
+        </dependency>
+
     </dependencies>
 
 </project>

--- a/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/pom.xml
+++ b/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/pom.xml
@@ -82,6 +82,11 @@
         </dependency>
 
         <dependency>
+            <groupId>jakarta.annotation</groupId>
+            <artifactId>jakarta.annotation-api</artifactId>
+        </dependency>
+
+        <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
             <scope>test</scope>

